src/Chapter2.hs

```diff
@@ -632,12 +631,9 @@ Write a function that takes elements of a list only on even positions.
 [2,3,4]
 -}
 takeEven :: [a] -> [a]
-takeEven lst = go (0::Int) lst
-  where
-    go _ [] = []
-    go n (x:xs)
-      | n `mod` 2 == 0 = x : go (n+1) xs
-      | otherwise      = go (n+1) xs
+takeEven [] = []
+takeEven (x:_:xs) = x : takeEven xs
+takeEven xs = xs 
```

Comment:
Great!
The first case is also covered by `take xs = xs` 🙂

Reply:
Good advice, fixed https://github.com/ericmoritz/learn4haskell/pull/2/commits/0327eab09f8f9edddd8418f5bfcbd6412e33b0f7